server/game/drawcard.js

```diff
@@ -207,13 +207,16 @@ class DrawCard extends BaseCard {
         return isNaN(baseSkillModifiers.baseMilitarySkill) || isNaN(baseSkillModifiers.basePoliticalSkill);
     }
 
+    /**
+     * Direct the skill query to the correct sub function.
+     * @param  {string} type - The type of the skill; military or political
+     * @return {number} The chosen skill value
+     */
     getSkill(type) {
-        /**
-         * Direct the skill query to the correct sub function.
-         * @param  {string} type - The type of the skill; military or political
-         * @return {integer} The chosen skill value
-         */
-        if(type === 'military') {
+        let skillFunction = this.mostRecentEffect(EffectNames.ChangeSkillFunction);
```

Comment:
I'm not sure that this is the best way to do this.  Shouldn't the engine be looking for ChangeSkillFunction in Conflict?

Reply:
I have wrapped the getSkill function with a getContributionToConflict function which will take this effect into consideration so the skill of the cards should not be muddied by the contribution to the conflict.  

(and also changed the name of the effect to `ChangeContributionFunction`)